lib/jobs/install-os.js

```diff
@@ -95,13 +95,21 @@ function installOsJobFactory(
                 }
             });
         }
-
         if (this.options.users) {
             _.forEach(this.options.users, function(user) {
                 assert.string(user.name);
                 assert.string(user.password);
                 assert.number(user.uid);
             });
+            this.context.users = this.options.users;
+        }
+        if (this.options.rootPassword || this.options.rootSshKey) {
```

Comment:
These codes are not suitable to put here, they don't matched with the function name `_validateOptions`, I hope to move them to another new function, or at least move to `_convertOptions` if you don't like to create a new function.